api/rolestore/client.go

```diff
@@ -490,10 +490,17 @@ func (store *RoleStore) ResolveUser(userID string) (*User, error) {
 
 // SearchUsers searches for users, matching the keywords and source
 // criteria.
-func (store *RoleStore) SearchUsers(searchBody UserSearchObject) ([]User, error) {
+func (store *RoleStore) SearchUsers(offset, limit int, sortkey, sortdir string, searchBody UserSearchObject) ([]User, error) {
 	result := usersResult{}
+	queryParam := Params{
```

Comment:
Everything looks good! Just one very minor thing, the variable name "queryParam" could be changed to "filters". There is absolutely nothing wrong with this variable name, I would just change it to keep the variable naming convention for the Params unified across the SDK.